HW_4/task_2.js

```diff
@@ -0,0 +1,56 @@
+/*
+1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив) 
+  и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+  Пиццы конкурента:
+  const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
+  
+2. Написать функцию, которая принимает предложение (слова разделенные только пробелами) в качестве параметра и выводит в консоль слово с наибольшим количеством букв. 
+  Если таких слов несколько - выводит их все.
+
+3. Напишите функцию, которая принимает на вход массив чисел, убирает из него дубликаты и возвращает массив с только уникальными значениями.
+
+4. Написать функцию, которая принимает на вход слово и проверяет, является ли это слово палиндромом
+
+*/
+
+function checkCompetitorPizzas(myPizzas){
+    const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai'];
+    let result = [];
+    for(pizza of competitorPizzas) {
+        if(!myPizzas.includes(pizza)){

```

Comment:
function doesn't check if the same pizza has different case. 
Try exp ['Peperoni', 'hawai', 'machette', 'caprioni', 'diablo']

Reply:
ok, but its a edge case. This task doesnt include Upper or Lower case